core/templates/components/state-editor/state-responses-editor/state-responses.component.html

```diff
@@ -82,7 +83,7 @@ <h3 class="oppia-exp-answer-card-header">Learner's Answers and Oppia's Responses
         </div>
         <div *ngIf="!suppressDefaultAnswerGroup()">
           <div class="oppia-option-list nav-stacked nav-pills" role="tablist">
-            <div class="option-list-default" [ngClass]="{'active': activeAnswerGroupIndex === answerGroups.length}">
+            <div class="option-list-default" [ngClass]="{'active': activeAnswerGroupIndex === answerGroups.length}" role="tab">
```

Comment:
Could you please break these attributes into their own individual lines (kinda like lines 23-28 above in this same file)?